File: karton-backend/app/Models/Inventory.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Inventory extends Model
{
    use HasFactory;

    protected $fillable = [
        'ingredient_id',
        'quantity',
        'minimum_level',
    ];

    public $timestamps = false;

    public function ingredient()
    {
        return $this->belongsTo(Ingredient::class);
    }
}

File: OrderMauiApp/Models/Inventory.cs
namespace LeltarSupportMauiApp.Models
{
    public class Inventory
    {
        public int IngredientId { get; set; }
        public decimal Quantity { get; set; }
        public decimal MinimumLevel { get; set; }

        public Ingredient? Ingredient { get; set; }
        public bool IsLowStock => Quantity <= MinimumLevel;
    }
}